bot.js

```diff
@@ -1,100 +1,137 @@
-const category = 'new';
-
-// supreme.seek(category, keywords, style, (product, err) => {
-//     if (err) {
-//         console.log(err);
-//         return err;
-//     }
-//     console.log(product);
-// });
 
-// grab and write to file for original invetnory
-// get current ALL inventory
-function getCurrentAllInventory(original) {
-	supreme.getItems('all', (product, err) => {
-	    if (err) {
-	        console.log(err);
-	        return err;
-	    }
-	    
-
-	    // if with original param then need to write a new history to file
-	    if (original) {
-	    	var json = JSON.stringify(product, null, 4);
-				fs.writeFile("og_inventory.json", json,'utf8', function(err) {
-				    if(err) {
-				        return console.log(err);
-				    }
-				    console.log("The file was saved!");
-				    return
-				}); 
-	    } else {
-	    	product.push({
-	        "title": "Supreme&#xAE;/The North Face&#xAE; Metallic Mountain Parka",
-	        "style": "Silver",
-	        "link": "http://www.supremenewyork.com/shop/jackets/sxjhcn3vb/dfkjdfd",
-	        "description": "Breathable, waterproof metallic coated nylon ripstop with fully sealed seams and all nylon faille weave at shoulders, elbows, and cuffs. Full zip closure with snap placket and three interior zippered pockets. Zippered alpine pockets and pit zips. Logo patches on right sleeve and embroidered logo at left chest and on back shoulder. Made exclusively for Supreme.",
-	        "addCartURL": null,
-	        "price": 388,
-	        "image": "https://assets.supremenewyork.com/149728/vi/Jk6XHIjRGvg.jpg",
-	        "sizesAvailable": null,
-	        "images": [
-	            "https://assets.supremenewyork.com/149728/zo/Jk6XHIjRGvg.jpg",
-	            "https://assets.supremenewyork.com/149729/zo/Dc6OSrcFF-o.jpg",
-	            "https://assets.supremenewyork.com/149730/zo/mZoRZlsnvt8.jpg"
-	        ],
-	        "availability": "Sold Out"
-    		}
-    ,)
-	    	comparisonFunc(product)
+// writes to a file
+function writeFile(filename, data) {
+	fs.writeFile(filename, data,'utf8', function(err) {
+	    if(err) {
+	        return console.log(err);
 	    }
-
-	});
+	    console.log("The file was saved!");
+	    return
+	}); 	
+};
+
+// saves all inventory (not slimmemd down json)
+// then writes to og_inventory.json
+function getCurrentAllInventory() {
+	supreme.getItems('all', (product, err) => {
+    if (err) {
+        console.log(err);
+        return err;
+    }
+  	var json = JSON.stringify(product, null, 4);
+  	writeFile("og_inventory.json", json)
+	})
+};
+
+// get current Slim inventory 
+function getCurrentSlimInventory(callback) {
+	supreme.getSlimItems('all', (slim_product, err) => {
+		if (err) {
+			console.log(err);
+			return err;
+		}
+// <----------------------/////////////////////////////////////------------------------------->
+
+		// for testing if function returns difference
+// 		slim_product.push(    {
+//     "title": "Supreme&#xAE;/The North Face&#xAE; Metallic Mountain Parka",
```

Comment:
Setup initial state w. array of objects to represent the "no new items" state. Set a random timer which will create a div and then append it to that object. The test returns true if your function correctly detects that something was added